app/forms/openid_connect_token_form.rb

```diff
@@ -122,6 +125,14 @@ def validate_aud_claim(payload)
                t('openid_connect.token.errors.invalid_aud', url: api_openid_connect_token_url))
   end
 
+  def validate_iat(payload)
+    iat = payload['iat']
+    return true if iat.blank?
```

Comment:
This line allows a blank IAT because it skips the error, right? I thought we wanted to require an IAT no matter what

Reply:
IAT isn't strictly required now, so it probably makes only return true here if there is no key, and then do the time/type validation following? I'm assuming we don't want to allow `nil`/`''`

Reply:
ok! I read a spec wrong https://github.com/18F/identity-idp/pull/4504/files#r540471553

Reply:
changed to be strict on integer in 62cd98641